py/packages/genkit/src/genkit/core/plugin_abc.py

```diff
@@ -23,30 +20,26 @@ class Plugin(abc.ABC):
     """
 
     @abc.abstractmethod
-    def attach_to_veneer(self, veneer: Genkit) -> None:
+    def initialize(self) -> None:
```

Comment:
this rename is based on @pavelgj comment that this function has more general purpose